docs/source/pillars/data_management.md

```diff
@@ -225,6 +225,11 @@ The ability of the {ref}`TRE operator <infrastructure_roles>` to identify, asses
   - Guidance
   - Importance
 * - 3.5.1.
+  - You should keep all TRE computing infrastructure up-to-date with security patches and antivirus (if appropriate).
```

Comment:
Are we saying antivirus should be kept up to date, or that we should also use that?

Reply:
Rephrased.